apps/phone/src/apps/twitter/components/AddTweetModal.tsx

```diff
@@ -80,8 +80,14 @@ const AddTweetModal = () => {
   const submitTweet = async () => {
     await promiseTimeout(200);
     const cleanedMessage = clean(message.trim());
-    if (cleanedMessage.length === 0) return;
-    if (!isValidMessage(cleanedMessage)) return;
+    if (cleanedMessage.length === 0) {
```

Comment:
You can just check if `cleanedMessage` is false instead of the length :)

Reply:
yeah I suppose i just left what was there from whoever wrote the code. ill change it now and push

Reply:
cleaned up, should be better